api/openapi.yaml

```diff
@@ -782,18 +815,10 @@ components:
         profilePicture:
           type: string
           readOnly: true
-        latestLocation:
-          $ref: '#/components/schemas/Coordinate'
-          readOnly: true
```

Comment:
I created the new endpoint to update the user latest location, this objects will be used only for the user profile functionality.

Reply:
The issue is that we need the latestLocation when we retrieve lists of users to show them on the map & their profiles (map: obvious; profile: we show the distance between the current user and the user whose profile they are viewing), which I think uses the same `User` schema from the OpenAPI spec.

Reply:
The front-end is the one providing with the current user location so I assume you should take it directly from the device.
Regarding the rest of the users, we have a separate schema which is light-weight and we will include the attribute there. which  is my next task.

Reply:
The current location of the user using the app is taken from the device, correct. But the endpoints `GET /users` (also used with the radius/area filter) and `GET /user/:id` use this schema and need to have the location. The former can be switched to the lightweight schema since we only need OnlineStatus and Location to display the users on the map, and additionally the Name and ProfileImage to show them in lists. But the latter is also used to show the profile (from the map and the full profile) and we also need the dogs and tags there, so they shouldn't be moved to the lightweight schema IMO.

Reply:
You lost me here...I do not see why we need a whole user object to be return when we call 'GET /users' and also why we need to have a location there.
('GET /user/:id') If you refer to the screen 'User overview' when a particular user is chosen, the functionality is not implemented yet. It is a separate task.

Reply:
I will add the coordinate to the User object when it will be required, if you don't mind.